### PR TITLE
increment required xcode_verison to 8.x

### PR DESCRIPTION
**Pull request by GenoZhou**
Related issue: #177 (Install by homework does not work)

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -42,8 +42,8 @@ end
 task :check_xcode_version do
   xcode_dir = `xcode-select -p`.chomp
   xcode_version = `mdls -name kMDItemVersion -raw "#{xcode_dir}"/../..`.chomp
-  unless xcode_version.start_with?('7.')
-    raise "\n[!!!] You need to use Xcode 7.x to compile SwiftGen. Use xcode-select to change the Xcode used to build from command line.\n\n"
+  unless xcode_version.start_with?('8.')
```
Comment (ikesyo) [inline Rakefile]: Until #171 or #176 is landed, this is not correct.

Comment (GenoZhou) [inline Rakefile]: I see. So any possible walkaround to install swiftgen 3.x with homebrew under Xcode 8?

Comment (arnonhongklay) [inline Rakefile]: ???? you solve #177 with change wording ?

**Comment by AliSoftware** [comment]
Fixed via #179

**Pull request closed without merging**